### PR TITLE
Remove unused dependencies

### PR DESCRIPTION
**Pull request by sroy3**
depcheks will still return:
```
Unused devDependencies
* @vscode/codicons
* eslint-plugin-n
* eslint-plugin-promise
* lint-staged
* prettier
```
Those are all used dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +60,6 @@
     "lint-staged": "14.0.1",
     "npm-run-all": "4.1.5",
     "nyc": "15.1.0",
-    "postcss": "8.4.31",
```
Comment (mattseddon) [inline package.json:64]: [Q] Does stylelint definitely not require this? @julieg18 was the one who added it, maybe she remembers

**Review by mattseddon — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sroy3) [reply package.json:64]: I'm not getting any errors or warnings when running `yarn lint:css`. I'll wait on @julieg18 to merge.

Comment (julieg18) [reply package.json:64]: I don't think stylelint requires it. It should work correctly without it. My guess is I accidentally added it when I was running get started scripts or I added it when I was troubleshooting.

**Review by julieg18 — APPROVED** [review]
Great work!

**Pull request merged**